content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -329,7 +329,7 @@ After the control plane is up, kubeadm completes the tasks described in followin
 
 ### (optional) Write base kubelet configuration
 
-{{< feature-state for_k8s_version="v1.11" state="beta" >}}
+{{< feature-state for_k8s_version="v1.22" state="deprecated" >}}
```

Comment:
Please help clarify whether the feature is deprecated or the gate flag is deprecated.
A feature deprecation means the dynamic kubelet config is no longer supported.
It smells to me that the feature is GA'ed rather than deprecated.

Reply:
Went through some more background about this...
It looks like the feature is gone. If that is the case, we are supposed to drop all related contents/sections/pages.

Reply:
I removed all the links to that page except from the features gates page and blog posts. Do you think it's time to remove the page content as well? I can remove it keeping only the caution note instead.